plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -61,6 +65,49 @@
 
     public static final String TYPE = "s3";
 
+    // prefix for aws client settings
+    private static final String PREFIX = "aws.config.";
```

Comment:
The link between the settings names and the logging class was never much use to me - I'd prefer meaningful, understandable names